### PR TITLE
added status indicator for plays

### PR DESCRIPTION
**Pull request by krispreza**

### CODE REVIEW

**Comment by jgkamat** [comment]
Overall, looks amazing, I need to do a full review though.

File: soccer/gameplay/main.py [hunk 1]
```diff
@@ -164,6 +164,13 @@ def save_playbook(file_name, isAbsolute=False):
         (PLAYBOOKS_DIR + '/' if not isAbsolute else '') + file_name,
         _play_registry.get_enabled_plays_paths())
 
+def clear():
+    global _play_registry
+    _play_registry.clear();
+
+def checkPlayStatus():
```
Comment (jgkamat) [inline soccer/gameplay/main.py]: I think this name is misleading, can we name it to something like `numEnabledPlays` or similar?

File: soccer/MainWindow.cpp [hunk 2]
```diff
@@ -949,6 +953,15 @@ void MainWindow::status(QString text, MainWindow::StatusType status) {
     }
 }
 
+void MainWindow::playStatus(bool color) {
```
Comment (jgkamat) [inline soccer/MainWindow.cpp]: I don't like the name `playStatus` either, it feels a bit too generic. 

Can we name it something like `playChangedStatus` or `playIndicatorStatus` or similar?

File: soccer/gameplay/GameplayModule.cpp [hunk 3]
```diff
@@ -255,6 +256,27 @@ void Gameplay::GameplayModule::savePlaybook(const string& playbookFile,
     PyGILState_Release(state);
 }
 
+void Gameplay::GameplayModule::clearPlays() {
+    PyGILState_STATE state = PyGILState_Ensure();
+    getMainModule().attr("clear")();
+    PyGILState_Release(state);
+}
+
+bool Gameplay::GameplayModule::checkPlayStatus() {
```
Comment (joshhting) [inline soccer/gameplay/GameplayModule.cpp]: I feel like this function could be named something that's more specific to what it does, like `playStatusChanged`?

File: soccer/gameplay/GameplayModule.cpp [hunk 4]
```diff
@@ -255,6 +256,27 @@ void Gameplay::GameplayModule::savePlaybook(const string& playbookFile,
...
+  PyGILState_STATE state = PyGILState_Ensure();
+  static int prevStatus = extract<int>(getMainModule().attr("checkPlayStatus")());
+  bool static change = false;
```
Comment (joshhting) [inline soccer/gameplay/GameplayModule.cpp]: Does this need to be static?

**Comment by chachmu** [comment]
@jgkamat Does this look good to you?

**Comment by jgkamat** [comment]
Yup, I'm good to go with this, we'll handle issues if they come up :P

**Pull request merged**